web/wp-content/themes/dctx-v2/inc/acf-search.php

```diff
@@ -0,0 +1,70 @@
+<?php
```

Comment:
:no_entry_sign: **Error**: Filenames should be all lowercase with hyphens as word separators. Expected phpcs-scan-2g259c.php, but found phpcs-scan-2g259C.php (*WordPress.Files.FileName.NotHyphenatedLowercase*).